Azure/sonic-buildimage
# [202012] [Arista] Update db with eeprom info through syseepromd (fixed regression)

#### Why I did it
Arista did not update db with eeprom info. Previous PR had issues that were reverted.

#### How I did it
Had Arista eeprom class inherit the class that has method to update db. Updated platform API methods for Arista 202012.

#### How to verify it
In redis-cli the keys and values can be seen. Can use sonic-mgmt testing to verify behavior, and see the chassis platform API methods have not regressed.

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog



#### A picture of a cute animal (not mandatory but encouraged)
